controllers/psychologists.js

```diff
@@ -1,93 +1,135 @@
-const fs = require('fs');
-const Psychologists = require('../data/psychologists.json');
+const Psychologists = require('../models/Psychologists');
 
-const getPsychologists = (req, res) => {
-  res.status(200).json(Psychologists);
+const availabilityObjectModel = {
+  monday: {
+    availability: '',
+    from: '',
+    to: '',
+  },
+  tuesday: {
+    availability: '',
+    from: '',
+    to: '',
+  },
+  wednesday: {
+    availability: '',
+    from: '',
+    to: '',
+  },
+  thursday: {
+    availability: '',
+    from: '',
+    to: '',
+  },
+  friday: {
+    availability: '',
+    from: '',
+    to: '',
+  },
+  saturday: {
+    availability: '',
+    from: '',
+    to: '',
+  },
+  sunday: {
+    availability: '',
+    from: '',
+    to: '',
+  },
 };
 
-const createPsychologist = (req, res) => {
-  const params = req.query;
-
-  const newPsychologist = {
-    id: new Date().getTime().toString(),
-    first_name: params.first_name,
-    last_name: params.last_name,
-    phone: params.phone,
-    email: params.email,
-    availability: JSON.parse(req.query.availability),
-    address: params.address,
-  };
-
-  Psychologists.push(newPsychologist);
-
-  fs.writeFile('./data/psychologists.json', JSON.stringify(Psychologists), {}, (error) => {
-    if (error) {
-      res.status(400).send(error);
-    }
-    return res.status(201).json(newPsychologist);
-  });
+const availabilityObjectAttrConstructor = (req) => {
+  const keys = Object.keys(availabilityObjectModel);
```

Comment:
what about using `const keys = Object.keys(req.body.availability || {});` instead of a mock?

Reply:
This is a great idea! I'll try it right now.